pkgs/os-specific/linux/igh-ethercat-master/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib, stdenv, fetchFromGitLab, autoconf, automake, pkg-config, libtool, kmod, iproute2, kernel }:
+
+stdenv.mkDerivation {
+  pname = "igh-ethercat-master";
+  version = "1.5.2-20221102-${kernel.modDirVersion}";
+
+  src = fetchFromGitLab {
+    owner = "etherlab.org";
+    repo = "ethercat";
+    rev = "2e2cef6131895336f87c57c18fe78ae01a90d3de";
```

Comment:
Are there no tags?

Reply:
Then version should be set to `unstable-2022-11-02` plus the kernel version.

Reply:
Agreed, fixed.